crypto/doc/nrf_cc310_mbedcrypto.rst

```diff
@@ -186,8 +185,9 @@ An alternative to allocating this on the heap is to provide a reference to a sta
 Usage restrictions
 ------------------
 
-On the nRF9160 SiP, the nrf_cc3xx_mbedcrypto library is restricted to only work in secure processing environment.
-The library uses mutexes to ensure single usage of hardware modules.
+The library can not be used in the non-secure domain of an application that uses ARM TrustZone.
+
+The library is thread-safe and can be invoked concurrently by multiple threads.
```

Comment:
The library isn't thread-safe unless you use the mutex integration which is provided on RTOS level. This happens in the nrf_cc3xx_platform library. There is nothing inside the library that makes it inherently thread-safe unless you give symbols for mutexes. 

Don't remove the point of single usage of hardware modules. For example: The symmetric engine in CryptoCell can only be used by one caller at the time, mutexes ensures this.

Reply:
fixed